ocp_install_env.sh

```diff
@@ -97,6 +97,19 @@ EOF
     fi
 }
 
+function additional_trust_bundle() {
+    if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ -z $MIRROR_IMAGES ]; then
+cat <<EOF
+additionalTrustBundle: |
+$(echo ${ADDITIONAL_TRUST_BUNDLE} | awk '{ print " ", $0 }')
+EOF
+    elif [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ ! -z "$MIRROR_IMAGES" ]; then
+cat <<EOF
+$(echo ${ADDITIONAL_TRUST_BUNDLE} | awk '{ print " ", $0 }')
+EOF
+    fi
```

Comment:
A less complicated version of this function:
```
if [ -n "${ADDITIONAL_TRUST_BUNDLE}" ]; then
    if [ -z "${MIRROR_IMAGES}" ]; then
        echo "additionalTrustBundle: |"
    fi
    echo "${ADDITIONAL_TRUST_BUNDLE}" | awk '{ print " ", $0 }'
fi
```

Reply:
Agree